common/src/main/java/com/gregtechceu/gtceu/integration/kjs/builders/machine/CustomMultiblockBuilder.java

```diff
@@ -342,6 +342,22 @@ public CustomMultiblockBuilder recipeModifier(BiFunction<MetaMachine, GTRecipe,
                 return this;
             }
 
+            public CustomMultiblockBuilder tier(int tier, Consumer<CustomMultiblockBuilder> consumer) {
+                for (var builder : builders) {
+                    if (builder.tier() == tier) {
+                        consumer.accept(builder);
+                    }
+                }
+                return this;
+            }
+
+            public CustomMultiblockBuilder allTiers(BiConsumer<Integer, CustomMultiblockBuilder> consumer) {
+                for (var builder : builders) {
+                    consumer.accept(builder.tier(), builder);
+                }
+                return this;
+            }
+
```

Comment:
use
```java
public CustomMultiblockBuilder tier(int tier, BuilderConsumer consumer);

@FunctionInterface
public interface BuilderConsumer {
    void accept(CustomMultiblockBuilder builder);
}
```
otherwise, probeJS couldnt recongnize the generic type.